### PR TITLE
Document change review process

### PR DESCRIPTION
**Pull request by joshuagl**
In-line with what is discussed in #130 here's a first stab at documenting our change review process. I don't know whether 5 business days is long enough a contemplation period, or whether the term "business days" is clear enough. I'd very much welcome feedback on those aspects, as well as the rest of the proposal.

In order to ensure it's clear _who_ will be doing the review, I've also added a `MAINTAINERS.md` listing the consensus builder and active TAP editors. Hopefully we can link to this document from [TAP 1](https://github.com/theupdateframework/taps/blob/master/tap1.md) so that it's clear to participants in the TAP process who the TAP Editors are.

Significant inspiration was taken from SPIFFE's [GOVERNANCE.md](https://github.com/spiffe/spiffe/blob/master/GOVERNANCE.md)

### CODE REVIEW

File: GOVERNANCE.md
Comment (joshuagl) [inline GOVERNANCE.md:47]: I wonder whether "two (2) weeks / ten (10) business days" would be a more appropriate consideration window.

Comment (mnm678) [reply GOVERNANCE.md:47]: I think 'weeks' is more universal than 'business days'. Holidays are usually just one day, so a week should still give everyone plenty of time to consider (with the possible exception of late December).

I think one week is plenty, we can always wait longer as needed.

File: GOVERNANCE.md
Comment (mnm678) [inline GOVERNANCE.md:38]: It might be good to clarify what 'state which can be merged' entails.

**Review by mnm678 — DISMISSED** [review]
A couple of minor comments, but this looks great.

File: GOVERNANCE.md
Comment (joshuagl) [reply GOVERNANCE.md:47]: I think you're right, especially as what days businesses are open on and engineers are working can vary by country.

File: GOVERNANCE.md
Comment (joshuagl) [reply GOVERNANCE.md]: I made an attempt at clarifying this without adding too much detail in 2844beb

File: GOVERNANCE.md
Comment (joshuagl) [reply GOVERNANCE.md]: I went ahead and changed it to "one (1) week" in 451aa7a

File: GOVERNANCE.md
Comment (trishankatdatadog) [inline GOVERNANCE.md:55]: for at (least|most) one week? What is the minimum and maximum waiting time?

Comment (joshuagl) [reply GOVERNANCE.md:55]: Good catch, thanks!

File: MAINTAINERS.md
Comment (joshuagl) [reply MAINTAINERS.md:7]: Done in f5b66a0

File: GOVERNANCE.md
Comment (joshuagl) [reply GOVERNANCE.md]: Fixed in 07b2b9e

**Comment by joshuagl** [comment]
I think we're all agreed on this, but in the spirit of the proposed changes I'd appreciate a second approval.

cc @JustinCappos @trishankatdatadog @lukpueh

**Review by trishankatdatadog — DISMISSED** [review]

**Comment by trishankatdatadog** [comment]
We should change the branch protection rules to require >= 2 reviews...

**Comment by lukpueh** [comment]
> We should change the branch protection rules to require >= 2 reviews...

Done.

**Comment by joshuagl** [comment]
Apologies, I forced push to squash in some of the fixup commits introduced during code review and that has dismissed the existing reviews.

Will need re-approval to meet the branch protection rules. Thank you in advance.

**Review by trishankatdatadog — APPROVED** [review]

**Review by mnm678 — APPROVED** [review]

**Comment by mnm678** [comment]
hmm, the Travis version checks are failing. This doesn't change the specification itself, so I'm going to go ahead and merge.

**Pull request merged**
